backend/src/API/teamEventsAPI.ts

```diff
@@ -135,3 +204,20 @@ export const deleteTeamEventAttendance = async (uuid: string, user: IdolMember):
   }
   await teamEventAttendanceDao.deleteTeamEventAttendance(uuid);
 };
+
```

Comment:
delete

Reply:
Shouldn't we keep the delete endpoint to allow users to cancel their requests? We can just keep the same code, but just remove the admin check.